ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+- PCSX2 version:
+``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
+
+
+- PCSX2 options:
+``(For example, Clamping/Rounding, Non-default settings (or) speedhacks used)``
```

Comment:
I'd add that it's preferred to have as much default settings as possible (I think that's preset "Safest" and MTVU disabled), and if there's no issue with these settings but there is an issue with some non default option, then state it clearly.